sw/device/silicon_creator/lib/crc32.c

```diff
@@ -26,6 +27,14 @@ void crc32_init(uint32_t *ctx) { *ctx = UINT32_MAX; }
  * lines 111-112 and 276-279.
  */
 void crc32_add8(uint32_t *ctx, uint8_t byte) {
+#ifdef OT_PLATFORM_RV32
+  uint32_t state = *ctx ^ byte;
+  __asm__("crc32.b %0, %1;"
+          : "=r"(state)
```

Comment:
I think this should be `+` instead of `=` because we're reading and writing the register.